nsv13/code/game/objects/items/custom_guns.dm

```diff
@@ -38,16 +38,16 @@
 
 /obj/item/projectile/bullet/peacekeeper
 	name = "6mm tungsten round"
-	damage = 10 //Hopefully making it bias towards armorpen should make it less likely to kill people straight out.
+	damage = 12 //Hopefully making it bias towards armorpen should make it less likely to kill people straight out. Minor buff to make it viable
 	armour_penetration = 30
 	icon = 'nsv13/icons/obj/projectiles_nsv.dmi'
 	icon_state = "pdc"
 
 /obj/item/projectile/bullet/peacekeeper/stun
 	name = "6mm electro-shock round"
-	damage = 6 //Straight up WAY worse than the security glock at stunning, but they serve a purpose
+	damage = 4 //Straight up WAY worse than the security glock at stunning, but they serve a purpose it used to kill a bit too often
 	armour_penetration = 0 //Less strong. Can't penetrate armor.
-	stamina = 15 //Slightly more than 3 bursts to stun
+	stamina = 20 //Slightly more than 3 bursts to stun Minor buff to make it viable
```

Comment:
Quite sad since that gun is horrible, but understandable. Values returned to original in [98bb89c].(https://github.com/BeeStation/NSV13/pull/2458/commits/98bb89c993757938cb0bea21b6e6ed411155d707)